repair/repair.cc

```diff
@@ -343,32 +343,35 @@ void tracker::done(repair_uniq_id id, bool succeeded) {
     }
     _done_cond.broadcast();
 }
-repair_status tracker::get(int id) const {
-    if (id >= _next_repair_command) {
+
+repair_status tasks::repair_module::get(int id) const {
+    if (id > _sequence_number) {
         throw std::runtime_error(format("unknown repair id {}", id));
     }
-    auto it = _status.find(id);
-    if (it == _status.end()) {
+    auto& tracker = _rs.repair_tracker();
+    auto it = tracker._status.find(id);
+    if (it == tracker._status.end()) {
```

Comment:
There seems to be duplication still between the repair_tracker status tracking and the repair tasks module.
Let's add a TODO to track the tasks entirely by the task module, and maybe even get rid of the repair tracker entirely, eventually.